webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/UserInformation.scala

```diff
@@ -67,7 +67,7 @@ class UserInformation extends DispatchSnippet with DefaultExtendableSnippet[User
       case Some(u) =>
         val displayName = userRepo.get(u.getUsername).runNow match {
           case None       => u.getUsername
-          case Some(info) => info.name.getOrElse(info.id)
+          case Some(info) => info.name.filter(_.nonEmpty).getOrElse(info.id)
```

Comment:
should test with `strip()` before `nonEmpty` no?

Reply:
Absolutely :see_no_evil: